app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -154,7 +153,7 @@ Check with [assessment only training providers](/train-to-be-a-teacher/assessmen
 
 [Supply teachers](https://www.tes.com/jobs/careers-advice/supply-teaching/how-become-supply-teacher) fill in when schools are temporarily short of staff. Register with a supply teacher agency for short-term roles in school.  
 
-[Find your local authority ('council')](https://www.gov.uk/find-local-council) and then search their website for teaching roles.
+[Find your local authority (council)](https://www.gov.uk/find-local-council) and then search their website for teaching roles.
 
 You could also consider [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) and education support roles. It can be helpful to gain experience of the English education system before you apply for teaching jobs. [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) lists these jobs. 
```

Comment:
```suggestion
You could also consider [working as a teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) or in education support roles. It can be helpful to gain experience of the English education system before you apply for teaching jobs. [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) lists these jobs. 
```